specification/automation/resource-manager/readme.md

```diff
@@ -162,7 +162,7 @@ go:
 
 ### Go multi-api
 
-``` yaml $(go)
+``` yaml $(go) && $(multiapi)

```

Comment:
Not sure how to review this, no idea where I can see what the `multiapi` tag does :)

Reply:
The multi-api tag should prevent `batch` from being invoked unless it was called for. Azure/azure-sdk-for-go#1023 is cause by multiple API Versions being generated when only a single one is called for.

Reply:
Is this the only config file that needs to be fixed?  I see batch all over the place.

Reply:
I did a search for all instances of `$(go)`, and didn't find any other instances where `batch` was unprotected by an accompanying `$(multiapi)`.